playbooks/group_vars/aio.yml

```diff
@@ -0,0 +1,37 @@
+---
+gating_overrides:
+  # (alextricity25) Adding the cirros_*_url vars. This var has been dropped
+  # from OSA in newton so we are carrying it here so we have support for both
+  # newton and earlier releases.
+  cirros_tgz_url: "http://download.cirros-cloud.net/{{ '{{' }} cirros_version {{ '}}' }}/cirros-{{ '{{' }}cirros_version{{ '}}' }}-x86_64-uec.tar.gz"
+  cirros_img_url: "http://rpc-repo.rackspace.com/rpcgating/cirros-0.3.4-x86_64-dropbearmod.img"
+  # (alextricity25) This variable is used to populate the image directives in
+  # tempest.conf in stable/newton.
+  # We can't rely on the upstream value of this variable to be the same as
+  # cirros_img_url, so we also override this variable here.
+  tempest_image_file: "cirros-0.3.4-x86_64-dropbearmod.img"
+  tempest_images:
+    - url: "{{ '{{' }}cirros_img_url{{ '}}' }}"
+      sha256: "ec1120a9310ac3987feee4e3c5108d5d0fd0e594c4283804c17d673ebb2d3769"
+    - url: "{{ '{{' }}cirros_tgz_url{{ '}}' }}"
+      sha256: "95e77c7deaf0f515f959ffe329918d5dd23e417503d1d45e926a888853c90710"
+  tempest_tempest_conf_overrides:
+    volume-feature-enabled:
+      snapshot: "True"
+
+  # Ensure raw_multi_journal is False for upgrades.
+  # This is because of the way migrate-yaml.py behaves with the
+  # '--for-testing-take-new-vars-only'; meaning that the new
+  # default variables will be set in the user_*_variables_overrides.yml
+  # file. Since raw_multi_journal is set to False as part of the deploy.sh
+  # process, but is set to True in Mitaka's
+  # user_rpco_user_variables_defaults.yml file, this will result in
+  # migrate-yaml.py adding 'raw_multi_journal: True' in the overrides.
+  # To avoid this behavior in gate, it is overridden here.
+  # The same is true for journal_size, and maas_notification_plan.
+  raw_multi_journal: "False"
+  journal_size: 1024
+  osd_directory: "True"
+
+maas_creds:
+  rackspace_cloud_auth_url: "https://identity.api.rackspacecloud.com/v2.0/"
```

Comment:
I think I have a misunderstanding on how the combining works. I want both `aio.yml` and `onmetal.yml` to inherit the MaaS credentials, seeing as they're the same. So, the only thing that should be injected further into that section is each deployment type's `rackspace_cloud_auth_url` variable.